icu4j/main/tests/core/src/com/ibm/icu/dev/data/numberformattestspecification.txt

```diff
@@ -353,6 +353,15 @@ minIntegerDigits	maxIntegerDigits	minFractionDigits	maxFractionDigits	output	bre
 // JDK fails here because it tries to use 9 + 6 = 15 sig digits.
 2	9	1	6	29.979246E7	K
 
+test ticket 20058
```

Comment:
There are two copies of numberformattestspecification.txt, which both need to be updated manually with the same patch (the other is in icu4c/source/test/testdata).  Please update the other copy as well as this copy.

Reply:
O. I updated icu4c/source/test/testdata/numberformattestspecification.txt. Please check.